components/phrase-display.tsx

```
@@ -3,11 +3,30 @@
+    value: number;
+    used: boolean;
+}
+
+const initialReveals: Reveals[] = Array.from({ length: 5 }, (_, i) => ({
+    value: i,
+    used: false
+}));
 
 const PhraseDisplay = () => {
 
     const [solution, setSolution] = useState<String>("");
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)